### DISCUSSION TITLE
How to get a count of the number of lines in one file and not the other

### DISCUSSION THREAD

**Discussion by smaring** [WinMerge/winmerge#1447] [Q&A]
I have two sorted lists with LDAP distinguished names containing thousands of lines each.  My goal is to figure out how many users are in one list but not the other.  I could certainly spend hours trying to count all the colored lines in one list, but that seems silly.  WinMerge already knows the line exists in one file but not the other.  Is there not a statistical comparison summary?

-Steve

**Comment by sdottaka** [comment]
> Is there not a statistical comparison summary?

Unfortunately there is no such functionality.

However, WinMerge has the ability to display only the lines with differences. You can display the lines with differences as shown below, copy the lines with differences using Ctrl + A and Ctrl + C keys, and paste them into another text editor to see the number of rows with differences.

![image](https://user-images.githubusercontent.com/98126/183907563-f9187d73-0f7a-4a0a-8ef6-24d84e80ba23.png)

**Comment by mischlrebl** [comment]
good description, thank you.

it is even enough to select the whole content - then the selection count is shown in the footer:
<img width="889" height="36" alt="image" src="https://github.com/user-attachments/assets/059364d7-f2c6-4b69-9ff2-41e3db1aeb70" />
